core/src/main/java/curso/modelo/Curso.java

```diff
@@ -0,0 +1,41 @@
+package curso.modelo;
+
+import curso.exception.ExceptionCamposObligatorios;
+import curso.exception.ExceptionFechaCierreIncorrecta;
+
+import java.time.LocalDate;
+
+public class Curso {
+    private String nombre;
+    private LocalDate fechaInicio;
+    private Nivel nivel;
+
+    private Curso(String nombre, LocalDate fechaInicio, Nivel nivel) {
+        LocalDate fechaActual = LocalDate.now();
+        if(nombre==null || nombre.trim().isEmpty() || fechaInicio==null || nivel==null){
+            throw new ExceptionCamposObligatorios("Error ,los campos no puede estar vacio");
```

Comment:
Colocar bien las comas
```suggestion
            throw new ExceptionCamposObligatorios("Error, los campos no puede estar vacio");
```